FSW/src/headers/commandHandling.hpp

```diff
@@ -18,13 +18,17 @@ bool checkIfCommandAllowed(int command);
 void executeAllCommands();
 void clearCommandQueue();
 void executeCommand(int command);
+void printInfo();
 
 namespace commandCode {
     // command codes are wrapped in a namespace so they are not global
     enum Code { // all possible commands
         // commands are 1 indexed so that the default initializer can be used for the command queue
+        // Ping
+        INFO = 1,                       // Triggers a readback of system data
+
         // Mode Change
-        ENTER_SAFE_MODE = 1,            // enter safemode state
```

Comment:
Make sure to update the docs if all these command numbers shifted. We ran into that today. Putting in the prevention from sending dangerous commands saved us, so good change there.

Reply:
Will do. I've been holding off since I've just been adding so many commands, but now that we are starting to test with the hardware I'll make sure to keep it up to date